plotting/scriptGenerator.py

```diff
@@ -56,6 +56,8 @@ def generateR(url, scriptType: str) -> BytesIO:
 
         if scriptType == "PLOT":
             template = template.format(q=query, var=var)
+        elif scriptType == "SUBSET":
+            template = template.format(q=query, init='{', end='}')
```

Comment:
You shouldn't need to specify `init='{', end='}' here. If you're getting an error is probably because of the `str.format()` function is confused by the for loop curly braces in the R template. I think the format will work if you double the braces. Something like this in the R template should work:
```
for (condition) {{
    do thing
}}
```

Reply:
Also, if you remove the `init='{', end='}'` you wont need the `elif scriptType == "SUBSET":` condition.